cmd/podman/cmd_varlink.go

```diff
@@ -0,0 +1,13 @@
+// +build varlink
+
+package main
+
+import (
+	"github.com/urfave/cli"
+)
+
+// getOptionalCommands returns an array of commands to be added to podman
+// when varlink is not specifically enabled, we return an empty array for now.
+func getOptionalCommands() []cli.Command {
```

Comment:
This approach won't scale well with additional optional commands behind different tags.  Perhaps we could have a `varlinkCommand` pointer that we setup in `varlink.go` in the `varlink` case and set `nil` in `varlink_dummy.go` (or some such) in the `!varlink` case?

Reply:
> This approach won't scale well with additional optional commands behind different tags.

I've filed baude/libpod#2 with a squash commit for this suggestion.